project/BuildDependencies/scripts/0_package.target-win10-win32.list

```diff
@@ -0,0 +1,42 @@
+; package.list should contain all file names of packages which are already in the "new" package format (the package contains the paths to the libraries
+; and headers relative to the XBMC root directory, stored in a zip container)
+; example sqlite-3.7.12.1-win32.7z:
+; -> (package name)\(relative path to file)
+; -> sqlite-3.7.12.1-win32\project\BuildDependencies\lib\sqlite3.lib
+; -> sqlite-3.7.12.1-win32\system\sqlite3.dll
+; -> ...
+;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
+cpluff-0.1.3-win10-Win32-v140.7z
+crossguid-fef89a4-win10-Win32-v140.7z
+curl-7.52.1-win10-Win32-v140.7z
+expat-2.2.0-win10-Win32-v140.7z
+fmt-3.0.1-win10-Win32-v140.7z
+freetype-2.8-win10-Win32-v140.7z
+lcms2-2.8-win10-Win32-v140.7z
+libaacs-0.9.0-win10-Win32-v140.7z
+libass-0.13.6-win10-Win32-v140.7z
+libbdplus-45c76dd-win10-Win32-v140.7z
```

Comment:
currently not in the win32 & x64 list

Reply:
Do you suggest to remove it?

Reply:
yes please, otherwise we wonder later why it is

Reply:
removed